lib/wiregasm/meson.build

```diff
@@ -42,10 +42,11 @@ link_args = [
   '-s', 'MODULARIZE=1',
   '-s', 'EXPORT_NAME="loadWiregasm"',
   '-s', 'TOTAL_MEMORY=128MB',
+  '-s', 'ENVIRONMENT=web',
```

Comment:
To generate a `Web` version without the need in `wg` and other node libraries.

Reply:
> Have you noticed any performance hits with the node check overheads in the generated lib?

The only inconvenience is in node js dependencies that needs to be installed to make the lib work with the bundlers. As during the bundling `emscripten` identifies the environment as NODE_ENIRONMENT.
 
> If we set it to `web`, it wouldn't work in any `nodejs` projects, even our tests won't run.

The flag is removed.

I also added tests to cover the new methods.

<img width="644" alt="Screenshot 2024-01-23 at 13 04 05" src="https://github.com/good-tools/wiregasm/assets/1247491/43251906-f93d-40e7-ab5a-bc5028e49b60">

Reply:
Thanks for adding the tests.

I think the bundling issues can be resolved by setting some overrides:

```js
config.resolve.fallback = {
  fs: false,
  path: false,
  crypto: false,
}
```

This is how the good.tools website uses it. Let me know if this works for you, or if you have a better alternative.

Reply:
> I think the bundling issues can be resolved by setting some overrides:

Thank you for sharing this. Adding the following lines to `package.json` helps Parcel to build w/o issues.

```json
  "alias": {
    "fs": false,
    "path": false,
    "ws": false,
    "crypto": false
  }
```